src/components/Discipline.vue

```diff
@@ -157,7 +168,7 @@ export default {
       {{ item.level }}
       <select
         class="discipline-ability-select"
-        :disabled="discipline.value < item.level ? 1 : 0"
+        :disabled="this.discipline.value + predatorDisciplineBonus < item.level  ? 1 : 0"
```

Comment:
tomuto uplne nerozumiem, asi mi tam niekde chybaju zatvorky...

Reply:
nebude to so zatvorkami menej prehladne?

Reply:
isAbility disabled method pridana, snad to je ok